app/components/atomic/galoy-slider-button/galoy-slider-button.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from "react"
+import { View } from "react-native"
+
+import { makeStyles, Slider, Text, useTheme } from "@rneui/themed"
+
+import { GaloyIcon } from "../galoy-icon"
+
+type GaloySliderComponentProps = {
+  disabled?: boolean
+  callback?: () => void
+  text?: string
+}
+
+enum sliderTextValues {
+  send = "slide to send",
+  sending = "send",
+  sent = "sent!",
+  failed = "failed",
+}
+
+const sliderStartValue = 6
+const sliderMaxValue = 100
+
+export const Sliders: React.FunctionComponent<GaloySliderComponentProps> = (props) => {
```

Comment:
Right now the slider is able to go backwards a little bit, can we prevent this?

Reply:
I haven't found a way around this. but it snaps back to its original position if taken backwards.